chat-plugins/poll.js

```diff
@@ -107,26 +135,39 @@ class Poll {
 		for (let i in this.room.users) {
 			let user = this.room.users[i];
 			if (user.userid in this.voters) {
+				// @ts-ignore
 				user.sendTo(this.room, `|uhtmlchange|poll${this.room.pollNumber}|${results[this.voters[user.userid]]}`);
 			} else if (user.latestIp in this.voterIps) {
+				// @ts-ignore
 				user.sendTo(this.room, `|uhtmlchange|poll${this.room.pollNumber}|${results[this.voterIps[user.latestIp]]}`);
 			}
 		}
 	}
 
-	updateTo(user, connection) {
-		if (!connection) connection = user;
+	/**
+	 * @param {User} user
+	 * @param {Connection?} [connection]
+	 */
+	updateTo(user, connection = null) {
+		const recipient = connection || user;
 		if (user.userid in this.voters) {
-			connection.sendTo(this.room, `|uhtmlchange|poll${this.room.pollNumber}|${this.generateResults(false, this.voters[user.userid])}`);
+			// @ts-ignore
+			recipient.sendTo(this.room, `|uhtmlchange|poll${this.room.pollNumber}|${this.generateResults(false, this.voters[user.userid])}`);
```

Comment:
Hm, why does this need a ts-ignore? Don't `connection` and `user` both have `sendTo` functions?

Reply:
The ignore is there because it complains about this.room.pollNumber, not the connection and user.